.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Not sure if I want to add vscode configurations.

Reply:
I thought it's maybe good to have it if someone uses codespaces to fix something quickly